app/src/main/java/com/spoony/spoony/presentation/explore/model/ExploreFilter.kt

```diff
@@ -43,31 +43,31 @@ object ExploreFilterDataProvider {
     }
     fun getDefaultRegionFilter(): ImmutableList<ExploreFilter> {
         val regionData = persistentListOf(
-            Pair(9, "강남구"),
-            Pair(10, "강동구"),
-            Pair(11, "강북구"),
-            Pair(12, "강서구"),
-            Pair(13, "관악구"),
-            Pair(14, "광진구"),
-            Pair(15, "구로구"),
-            Pair(16, "금천구"),
-            Pair(17, "노원구"),
-            Pair(18, "도봉구"),
-            Pair(19, "동대문구"),
-            Pair(20, "동작구"),
-            Pair(21, "마포구"),
-            Pair(22, "서대문구"),
-            Pair(23, "서초구"),
-            Pair(24, "성동구"),
-            Pair(25, "성북구"),
-            Pair(26, "송파구"),
-            Pair(27, "양천구"),
-            Pair(28, "영등포구"),
-            Pair(29, "용산구"),
-            Pair(30, "은평구"),
-            Pair(31, "종로구"),
-            Pair(32, "중구"),
-            Pair(33, "중랑구")
+            Pair(1, "강남구"),
+            Pair(2, "강동구"),
+            Pair(3, "강북구"),
+            Pair(4, "강서구"),
+            Pair(5, "관악구"),
+            Pair(6, "광진구"),
+            Pair(7, "구로구"),
+            Pair(8, "금천구"),
+            Pair(9, "노원구"),
+            Pair(10, "도봉구"),
+            Pair(11, "동대문구"),
+            Pair(12, "동작구"),
+            Pair(13, "마포구"),
+            Pair(14, "서대문구"),
+            Pair(15, "서초구"),
+            Pair(16, "성동구"),
+            Pair(17, "성북구"),
+            Pair(18, "송파구"),
+            Pair(19, "양천구"),
+            Pair(20, "영등포구"),
+            Pair(21, "용산구"),
+            Pair(22, "은평구"),
+            Pair(23, "종로구"),
+            Pair(24, "중구"),
+            Pair(25, "중랑구")
```

Comment:
p3
매번 Index 붙이는거 너무너무 귀찮은데 아래처럼 Index를 활용하는건 어때요?

```suggestion
val regionNames = listOf(
    "강남구", "강동구", "강북구", "강서구", "관악구", "광진구", "구로구", "금천구", "노원구",
    "도봉구", "동대문구", "동작구", "마포구", "서대문구", "서초구", "성동구", "성북구",
    "송파구", "양천구", "영등포구", "용산구", "은평구", "종로구", "중구", "중랑구"
)

val regionData = regionNames.mapIndexed { index, name ->
    Pair(index + 1, name)
}.toPersistentList()

```

Reply:
해당 부분은 추후에 (장소 리스트 가져오기 API) 붙이면서 그때 수정하도록 하겠습니다. 감사합니다!